packages/core/src/events/mouse.ts

```diff
@@ -1402,10 +1402,6 @@ export function handleContextMenu(
   // }
   const workbookRect = workbookContainer.getBoundingClientRect();
 
-  if ((ctx.luckysheet_select_save?.length ?? 0) === 0) {
-    return;
-  }
```

Comment:
Why delete this?

Reply:
We can change selection and open the contextMenu by right-click a cell out of selection now.
If we stay this, we can't use this feature when there is no selection.